packages/consumption/test/modules/attributes/AttributesController.test.ts

```diff
@@ -2133,10 +2213,8 @@ describe("AttributesController", function () {
                 }
             });
 
-            await TestUtil.expectThrowsAsync(
-                consumptionController.attributes.getSharedVersionsOfRepositoryAttribute(relationshipAttribute.id),
-                "error.consumption.attributes.invalidPropertyValue"
-            );
+            const result = await consumptionController.attributes.getSharedVersionsOfRepositoryAttribute(relationshipAttribute.id);
```

Comment:
Why should this function no longer throw an error if no RepositoryAttribute ID is used?